src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -178,6 +179,38 @@ public Settings GetSettings()
             return _settings;
         }
 
+        /// <summary>
+        /// Gets the back/forward list object of this view.
+        /// </summary>
+        /// <returns>The back/forward list object of this view.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardList GetBackForwardList()
+        {
+            if (_backforwardlist == null)
+            {
+                IntPtr backforwardlistHandle = Interop.ChromiumEwk.ewk_view_back_forward_list_get(_realHandle);
+                if (backforwardlistHandle == IntPtr.Zero)
+                {
+                    return null;
+                }
+                _backforwardlist = new BackForwardList(backforwardlistHandle);
+            }
+            return _backforwardlist;
+        }
+
+        /// <summary>
+        /// Clear the back/forward list object of this view.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void ClearBackForwardList()
```

Comment:
```suggestion
        public void Clear()
```

Reply:
WebView.Clear() is ambiguous, I think that WebView.ClearBackForwardList() is better.

Reply:
I can do this, but
In BackForwardList.cs, i will be using ewk_view APi which is supposed to be in WebView,cs

Reply:
In my opinion, it doesn't matter

Reply:
OK I will change

Reply:
Actually `BackForwordList` is just a snapshot at a moment in time. So clearing of the snapshot is meaningless.
I think if we provide `BackForwordList.Clear`, developer can misunderstand that `BackForwordList` is connected with the real data.

Reply:
if `BackForwordList` is snapshot of real data,
it should be provided as method instead of property

A list type property is provided as a live data not snapshot.

Reply:
OK, we will provide it as a method. Thanks for the answer!